eahub/profiles/models.py

```diff
@@ -68,6 +68,75 @@ class GivingPledge(enum.Enum):
         ONE_FOR_THE_WORLD: "One for the World",
     }
 
+class OrganisationalAffiliations(enum.Enum):
+
+    EIGHTYTHOUSAND_HOURS = 1
```

Comment:
Nit: `EIGHTY_THOUSAND` would be closer to standard English than `EIGHTYTHOUSAND`.